app-admin/fleet/metadata.xml

```diff
@@ -3,7 +3,7 @@
 <pkgmetadata>
   <!-- maintainer-needed -->
   <longdescription lang="en">
-	</longdescription>
+    </longdescription>
```

Comment:
Feel free to fix it but via removing the empty element altogether. Spaces are not very useful for a description anyway ;-).